cheroot/workers/threadpool.py

```diff
@@ -180,8 +183,16 @@ def put(self, obj):
                 waiting to be processed
         """
         self._queue.put(obj, block=True, timeout=self._queue_put_timeout)
-        if obj is _SHUTDOWNREQUEST:
-            return
+
+    def _clear_dead_threads(self):
+        # Remove any dead threads from our list
+        for t in self._threads:
+            if not t.isAlive():
```

Comment:
Could you please change this to a "guard expression" so that the nesting level would be lower?
```suggestion
            if t.isAlive():
                continue
```

Reply:
I used your later suggestion.